org.mixedrealitytoolkit.uxcore/Experimental/List/VirtualizedScrollRectList.cs

```diff
@@ -531,6 +524,22 @@ public void SetItemCount(int newCount)
             visibleValid = false;
             UpdateScrollView(scroll);
         }
+
+        /// <summary>
+        /// Resets the VirtualizedScrollRectList. This may remove items aready
+        /// visible and create new items.
+        /// </summary>
+        public void Reset()
```

Comment:
Reset() is a MonoBehaviour editor mode event function.   Can you please rename this metho to avoid confusion.   Perhaps `ResetLayout`?

Reply:
It is now called reset layout